src/content/docs/en/guides/deploy/cloudflare.mdx

```diff
@@ -137,3 +137,14 @@ If you're encountering errors, double-check the version of `node` you're using l
 Cloudflare requires [Node v16.13](https://miniflare.dev/get-started/cli#installation), which is a more recent version than Astro’s out-of-the-box minimum, so double check you’re using at least v16.13.
 
 Client-side hydration may fail as a result of Cloudflare's Auto Minify setting. If you see `Hydration completed but contains mismatches` in the console, make sure to disable Auto Minify under Cloudflare settings.
+
+If you are building a project that is using server-side rendering and the server fails to build with error messages similar to:
+
+`[Error] Could not resolve "XXXX` `The package "XXXX" wasn't found on the file system but is built into node. Are you trying to
+  bundle for node? You can use "platform: 'node'" to do that, which will remove this error.`
+
+This means that a package or import you are using in the server-side environment is not compatible with the [Cloudflare Workers runtime APIs](https://developers.cloudflare.com/workers/runtime-apis/nodejs/). 
+
+If you are directly importing a Node.js runtime API, please refer to the Astro documentation on Cloudflare's [Node.js compatibility](#nodejs-compatibility) for further steps on how to resolve this.
```

Comment:
Oh my bad.. I actually might have be wrong here.. this is not the integration page, but the deploy guide 😮‍💨